.github/workflows/pkg_msi.yaml

```diff
@@ -53,12 +53,17 @@ jobs:
           echo "trimmed_version=$(echo ${V} | sed 's/-.*//')" >> $GITHUB_OUTPUT
       - name: Ensure version of cnquery and cnspec are available
         run: |
-          curl -sL --head --fail https://github.com/mondoohq/cnquery/releases/download/v${{ steps.version.outputs.version }}/cnquery_${{ steps.version.outputs.version }}_windows_amd64.zip
-          curl -sL --head --fail https://github.com/mondoohq/cnspec/releases/download/v${{ steps.version.outputs.version }}/cnspec_${{ steps.version.outputs.version }}_windows_amd64.zip
+          curl -sL --head --fail https://github.com/mondoohq/cnquery/releases/download/v11.26.0/cnquery_11.26.0_windows_amd64.zip \
```

Comment:
why do I feel this command went from being parameterized to be hardcoded? 🤔

Reply:
oops! fixed 👍